app/containers/InAppNotification/index.tsx

```diff
@@ -30,7 +33,15 @@ const InAppNotification = memo(() => {
 		const state = Navigation.navigationRef.current?.getRootState();
 		const route = getActiveRoute(state);
 		if (payload?.rid || notification.customNotification) {
-			if (payload?.rid === subscribedRoom || route?.name === 'JitsiMeetView' || payload?.message?.t === 'videoconf') return;
+			if (route?.name === 'JitsiMeetView' || payload?.message?.t === 'videoconf') return;
```

Comment:
App is going to vibrate no matter which thread I'm in or which thread triggered the notification.
- If I'm on main room and receive it, it's ok, because there's the visual thread badge indicator.
- If I'm on a thread and receive on main room or another thread, there's nothing linking the vibration to the new message.

We know what it is for obvious reasons, but it might be confusion for the users. It might be considered a trivial bug or even the expected behavior, but it's a valid point to be said here. I'd stick to the current version and get feedback. If necessary, we try a different approach.

Reply:
I take in consideration what you said and I refactored the code. Thanks for pointing.